src/common/buttons/button.module.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../assets/stylesheets/colors.scss';
+
+.customButton {
+  background-color: $sky-blue;
+  height: 3em;
+  border-radius: 10px;
+  width: 40%;
+  color: white;
+  box-shadow: $sky-blue 0px 7px 29px 0px;
+}
+
+.customButton:hover {
+  background-color: $sky-blue-on-hover;
+  width: 41%;
+}
+
+.customButton:disabled {
```

Comment:
because you're using `SCSS` you could  nest these classes as:

```
.customButton {
// button classes

    &:hover {
            // hover classes
    }

    &:disabled {
    // disabled classes
    }
}
```